Source/Krypton Components/Krypton.Toolkit/General/OutlookGrid/General/OutlookGridDefaultGroup.cs

```diff
@@ -339,11 +339,11 @@ public virtual object Clone()
         /// </summary>
         /// <param name="obj">the value in the related column of the item to compare to</param>
         /// <returns></returns>
-        public virtual int CompareTo(object obj)
+        public virtual int CompareTo(object? obj)
         {
             int orderModifier = Column.SortDirection == SortOrder.Ascending ? 1 : -1;
             int compareResult = 0;
-            object? o2 = ((OutlookGridDefaultGroup)obj).Value;
+            object? o2 = (obj as OutlookGridDefaultGroup)!.Value;
```

Comment:
`o2` is being ignored nullable when used, so check before usage and exit / throw

Reply:
changed to:
`object? o2 = (obj as OutlookGridDefaultGroup)?.Value;`

Logic handles the rest:
```cs
            if ((_val == null || _val == DBNull.Value) && o2 != null && o2 != DBNull.Value)
            {
                compareResult = 1;
            }
            else if (_val != null && _val != DBNull.Value && (o2 == null || o2 == DBNull.Value))
            {
                compareResult = -1;
            }
            else 
            {
                 . . . .
            )
```